sympy/vector/vector.py

```diff
@@ -345,6 +344,9 @@ def separate(self):
 class BaseVector(Vector, AtomicExpr):
     """
     Class to denote a base vector.
+
+    Unicode pretty forms in Python 2 should use the `u` prefix.
```

Comment:
@asmeurer Both the changes have been made. I guess this can be merged as soon as the tests pass.